components/leave_private_channel_modal/leave_private_channel_modal.tsx

```diff
@@ -63,14 +93,31 @@ export default class LeavePrivateChannelModal extends React.Component {
     handleHide = () => {
         this.setState({
             show: false,
-            channel: null,
+            channel: {
+                id: '',
+                display_name: '',
+                type: 'P',
+                team_id: '',
+                name: '',
+                header: '',
+                purpose: '',
+                creator_id: '',
+                scheme_id: '',
+                group_constrained: false,
+                create_at: 0,
+                update_at: 0,
+                delete_at: 0,
+                last_post_at: 0,
+                total_msg_count: 0,
+                extra_update_at: 0
+            }
         });
     };
 
     render() {
-        let title = '';
-        let message = '';
-        if (this.state.channel) {
+        let title;
+        let message;
+        if (this.state.channel.display_name) {
```

Comment:
I think this should be:
```suggestion
        if (this.state.channel && this.state.channel.display_name) {
```